bin/PerlDDS/Run_Test.pm

```diff
@@ -464,7 +464,132 @@ sub wait_kill {
 
   my $process = $self->{processes}->{process}->{$name}->{process};
 
-  return PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+  my $result = PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+
+  # If opts has indicated that user wants to get core dump/signal information
+  # then find the core dump file if any and print its stack trace.
+  if (defined $opts && defined $opts->{dump_ref} && ${$opts->{dump_ref}}) {
```

Comment:
Also, does this need to check for SEGV or other error condition? I may be misunderstanding the Perl, but it looks like this will always run (even on success?).

Reply:
So if there is a core dump, generated by SEGV or some other signal, the body of the `if` block will run. But as in the previous comment, we can make it independent of what user sets.

Reply:
I'm wondering what will happen if we have a successful run but there happen to be old core files laying around and the pattern doesn't exclude them... for example if the pattern is just "core" or "core.%e" etc.

Reply:
`${$opts->{dump_ref}}` indicates if the process has failed and had a core dump. So it won't try finding a core dump if the process succeeded.